no-for-apaertheid.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: No Tech 4 Apartheid
+permalink: /no-tech-for-apartheid/
+---
+Call to Action for Tech Workers
+
+“I work in tech and I am so sad/angry/scared about the current crisis in Gaza, but no one in tech seems to be talking about it.”
```

Comment:
Can you please replace all of this content w what's in the doc now?

Section headers should be prefixed w `##`.

Omit the bracketed paragraphs & add links?

Reply:
Oh & need to add links.